### ISSUE TITLE
MO2 unlocking - usvfs waited for chrome.exe before completing

### ISSUE THREAD

**Issue by bartmoss** [ModOrganizer2/modorganizer#507]
# The problem

MO2 waited for chrome.exe to end before unlocking.

# Environment

* Mod Organizer Version that exhibits the issue: 2.1.5
* Last Mod Organizer Verison that did not exhibit the issue (if applicable): Unknown
* Desktop OS/version used to run Mod Organizer: Windows 10

# Details

I've not done anything special, as far as I am aware.

- I launched the SSE CK through MO2
- I edited a small mod I am working on
- I quit the SSE CK normally
- Found that MO2 didn't unlock
- Checked the task manager to see if Skyrim or the CK was still showing in the task list; this was not the case
- Noticed that the log pane of the MO2 window said waiting for usvfs process completion: chrome.exe (21208)
- Quit all Chrome windows and tabs
- This unlocked MO2

Unfortunately I did not check if any of my chrome tabs had any relevant website names (in case there's some odd name matching going on); I did google a small CK issue I had.

I should add that this is the first version of MO2 I use "in production". I had launched the CK twice before earlier today, and it caused no issues. I also *think* that MO2 did not unlock after I had launched SkyrimSE itself, earlier, and manually unlocked. I paid it no attention at the time though.

# Link to Mod Organizer logs

## USVFS
https://gist.github.com/bartmoss/0365711b5ac504259c9c623e1834e469

## MO Interface
https://gist.github.com/bartmoss/8fab9ae1c9b7ecfa0eab0a80ba0f7bd9

**Comment by Project579** [comment]
That is chrome's malware scanner fault.

**Comment by Project579** [comment]
It scans your enitire PC and so gets caught up into MO2 virtual folder.

**Comment by bartmoss** [comment]
Hmm. So the virtual folder detects if a program accesses files in it; and I assume it can't know if that software will attempt to access more files, so it waits for the process to shut down for good. Correct?

**Comment by Project579** [comment]
Yes in the future we will probably add a Blacklist to limit these kinds of problems.

**Comment by Al12rs** [comment]
Implemented backlist by LostDragonist as of https://github.com/ModOrganizer2/usvfs/commit/7f0eda57fa49b5fb3e346e690de9ac700cb11ae1
